### PR TITLE
Feature/tdi 35765 jira components (WIP)

### PR DESCRIPTION
**Pull request by gonchar-ivan**

### CODE REVIEW

File: components-jira/pom.xml [hunk 1]
```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.talend.components</groupId>
+		<artifactId>components-parent</artifactId>
+		<version>0.7.0-SNAPSHOT</version>
```
Comment (sgandon) [inline components-jira/pom.xml]: use version 0.8.0-SNAPSHOT

File: components-jira/pom.xml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+	</parent>
+
+	<properties>
+		<components.api.version>0.7.0-SNAPSHOT</components.api.version>
```
Comment (sgandon) [inline components-jira/pom.xml]: use version 0.8.0-SNAPSHOT

File: components-jira/src/main/java/org/talend/components/jira/connection/Rest.java [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    public String post(String resource, String body) throws ClientProtocolException, IOException {
+        Request post = Request.Post(hostPort + resource).bodyString(body, contentType);
+        for (Header header : headers) {
+            post.addHeader(header);
+        }
+        Content content = post.execute().returnContent();
+        if (content != null) {
+            return content.asString();
+        }
+        return "204 no content";
```
Comment (sgandon) [inline components-jira/src/main/java/org/talend/components/jira/connection/Rest.java]: This is a strange return, why don't you return null ?

**Comment by gonchar-ivan** [comment]
Fixed according Sebastian's comments

**Pull request merged**